### DISCUSSION TITLE
Is possible use docker-compose ?

### DISCUSSION THREAD

**Discussion by domenicomonaco** [spatie/docker#45] [Ideas]
Spatie can start an docker-compose yml ?

**Comment by inmanturbo** [comment]
In a way this package is actually an alternative to docker-compose. It's an abstraction layer around docker commands using symfony process, but this one only deals with one instance at a time so it would not work with compose. If you want to call docker-compose from php programmatically, you can simply call `docker-compose` using the  symfony process component. I'm on a mobile device right now and cannot provide links, so please Google "symfony process component".
